src/components/ImageUploader/index.tsx

```diff
@@ -20,7 +20,13 @@ export interface UploaderProps {
 
 /* ImageUploader Props */
 export type ImageUploaderProps = {
+  /** image uploader로 업로드된 이미지의 목록을 정합니다.
+   * @type ImageInfo[]
+   */
   images: ImageInfo[]
+  /** image uploader의 onChange 이벤트 시, 호출될 함수를 정합니다.
+   * @type UploaderOnChangeHandler
+   */
```

Comment:
컴포넌트 이름으로 통일하는게 어떨까용 ? 
```suggestion
  /** ImageUploader로 업로드된 이미지의 목록을 정합니다.
   * @type ImageInfo[]
   */
  images: ImageInfo[]
  /** ImageUploader의 onChange 이벤트 시, 호출될 함수를 정합니다.
   * @type UploaderOnChangeHandler
   */
```